File: Php/app/Player/Listing/PlayerItemModel.php
namespace app\Player\Listing{

	use app\Urls\PlayerDetailsUrlModel;
	use entities\Homegame;
	use entities\Player;

	class PlayerItemModel{

		public $name;
		public $urlModel;
		public $email;

		public function __construct(Homegame $homegame, Player $player){
			$this->name = $player->getDisplayName();
			$this->urlModel = new PlayerDetailsUrlModel($homegame, $player);
		}

	}

}

File: Web/Models/PlayerModels/Listing/PlayerItemModel.cs
using Core.Classes;
using Web.Models.UrlModels;

namespace Web.Models.PlayerModels.Listing{

	public class PlayerItemModel{

	    public string Name { get; set; }
        public UrlModel UrlModel { get; set; }
	    public string Email { get; set; }

		public PlayerItemModel(Homegame homegame, Player player){
			Name = player.DisplayName;
			UrlModel = new PlayerDetailsUrlModel(homegame, player);
		}

	}

}